roles/ovirt-datacenter-cleanup/tasks/find_hosted_engine_vm_cluster.yml

```diff
@@ -0,0 +1,5 @@
+- name: Find cluster that contains the hosted engine VM
```

Comment:
There is nice filter called `selectattr`[1] maybe you can find it usefull in this use case.

Something like this could work:
```yaml
"{{ ovirt_vms  | selectattr("name", "equalto", hosted_engine_vm_name) }}
```

[1] http://jinja.pocoo.org/docs/dev/templates/#selectattr

Python jina 2.7 should be available on el7, so it shouldn't cause any issues.